pkgs/desktops/lomiri/development/lomiri-app-launch/default.nix

```diff
@@ -58,6 +58,13 @@ stdenv.mkDerivation (finalAttrs: {
       hash = "sha256-11pEhFi39Cvqb9Hg47kT8+5hq+bz6WmySqaIdwt1MVk=";
     })
 
+    # Remove when https://gitlab.com/ubports/development/core/lomiri-app-launch/-/merge_requests/65 merged & in release
```

Comment:
Please download and vendor the patch.

There is rising concerns about the availability of the URL, if the patch is in nixpkgs directly this is not a problem.

Reply:
Any issue or discussion threads on this? Pretty sure we used to *not* vendor patches when they could instead be sent to upstream & fetched from there.

Reply:
There's a section in our contributing document specifically for this: https://github.com/NixOS/nixpkgs/blob/2c8f8a719a9018a43984062272ebf5772dc786d6/pkgs/README.md#vendoring-patches

> In the following cases, a `.patch` file _should_ be added to Nixpkgs repository, instead of retrieved:
> 
> […]
> 
> - is already proposed upstream but not merged yet

Just linking that would've been enough to clear that up for me.

---

Since then, both MRs have been accepted & merged, so this is all not relevant for this PR anymore.